sdk/nodejs/__tests__/cloudClient.spec.ts

```diff
@@ -17,384 +17,396 @@ const respond500User = { user_id: '500' }
 
 jest.mock('fetch-retry')
 
-describe('DevCycleCloudClient without EdgeDB', () => {
+describe('DevCycleCloudClient', () => {
```

Comment:
these tests were flaking due to some kind of race condition with shutting down the mock service worker while trying to start it again